lib/commands/create.js

```diff
@@ -6,6 +6,10 @@ import {logger} from '../logger';
 import shelljs from 'shelljs/shell';
 
 export default function create(appPath) {
+  if (typeof appPath === 'undefined' || appPath === '') {
```

Comment:
Oh, I used that as a precaution to avoid multiple blank spaces (such as appPath = "  ")